instrumentation/apache-httpclient/apache-httpclient-5.2/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v5_2/ApacheHttpClient5Request.java

```diff
@@ -124,13 +72,4 @@ private static URI getCalculatedUri(HttpHost httpHost, URI uri) {
       return null;
     }
   }
-
-  @Nullable
-  public InetSocketAddress getServerSocketAddress() {
```

Comment:
It seems many methods in this class was copied from other version modules, delete them together? such as https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/d156afb48bbbc642785b64c2ae338bee1b6b5876/instrumentation/apache-httpclient/apache-httpclient-4.3/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v4_3/ApacheHttpClientRequest.java#L129

Reply:
this method is used in 4.3 instrumentation